aggregator/src/recursion/circuit.rs

```diff
@@ -68,21 +76,39 @@ fn accumulate<'a>(
 
 #[derive(Clone)]
 pub struct RecursionCircuit<ST> {
+    /// The verifying key for the circuit.
     svk: Svk,
+    /// The default accumulator to initialise the circuit.
     default_accumulator: KzgAccumulator<G1Affine, NativeLoader>,
+    /// The SNARK witness from the k-th BatchCircuit.
     app: SnarkWitness,
+    /// The SNARK witness from the (k-1)-th BatchCircuit.
```

Comment:
from the previous RecursionCircuit, i.e. the RecursionCircuit which has aggregated (k-1) snarks from BatchCircuit